src/connection/kernel_connection.rs

```diff
@@ -2,12 +2,14 @@
 use std::{
     fs::{File, OpenOptions},
     io::{Read, Write},
+    os::fd::AsFd,
     path::Path,
 };
 
+use nix::poll::{poll, PollFd, PollFlags, PollTimeout};
 use tracing::debug;
 
-use super::{PrinterConnection, printer_connection::sealed::ConnectionImpl};
+use super::{printer_connection::sealed::ConnectionImpl, PrinterConnection};
```

Comment:
I noticed this in some other files as well - you might want to run `rust fmt` on the codebase @mkienitz (and possibly enable format-on-save in your editor).